### PR TITLE
feat: add new snippets

### PR DESCRIPTION
**Pull request by m1ga**
Just some Alloy snippets to quickly generate a ListView, NavigationWindow, TabGroup and OptionDialog alloy preset.

Saves some time :wink: 

* removed the atom parts from the github action to make it :green_circle: again
* some quick "Appc" -> "Titanium" renaming

### CODE REVIEW

File: snippets/alloy.cson [hunk 1]
```diff
@@ -66,4 +66,43 @@
...
+        'prefix': 'ListView'
+        'body': """
+        	<ListView id="" defaultItemTemplate="default">
+        		<Templates>
+        			<ItemTemplate name="default"></ItemTemplate>
+        		</Templates>
+        		<ListSection>
+        			<ListItem/>
+        		</ListSection>
+        	</ListView>
```
Comment (ewanharris) [inline snippets/alloy.cson:81]: What do you think about using some [placeholders](https://macromates.com/manual/en/snippets#placeholders)/[tab stops](https://macromates.com/manual/en/snippets#tab_stops) in these snippets? We'd provide defaults and then folks can tab through them and edit as necessary

So for example
```suggestion
        	<ListView id="${1:listview}" defaultItemTemplate="${1:default}">
        		<Templates>
        			<ItemTemplate name="${1:default}"></ItemTemplate>
        		</Templates>
        		<ListSection>
        			<ListItem/>
        		</ListSection>
        	</ListView>
```

**Comment by m1ga** [comment]
Quickly removed some other "Appcelerator Titanium" labels too

**Comment by m1ga** [comment]
I've removed the atom parts from the action, so it's green again :wink:

**Review by ewanharris — APPROVED** [review]
Pushed an extra commit to just remove the test step altogether

**Pull request merged**